circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
You mean Circle might not set the env variable until after it runs the tests? I put this here just for testing, so it would fail fast.